docs/01_introduction/README.md

```diff
@@ -143,60 +143,60 @@ Some commonly used optional easyconfig parameters include:
 * ``easyblock``, which specifies which (generic) easyblock should be used;
 * ``sources`` and ``source_urls``, which specify the list of source files and where to download them;
 * ``dependencies`` and ``builddependencies``, which specify (drum roll...) the list of (build) dependencies;
-* ``configopts``, ``buildopts``, and ``installopts``, which speficy options for the configuration/build/install commands, resp.;
+* ``configopts``, ``buildopts``, and ``installopts``, which specify options for the configuration/build/install commands, respectively;
+
+If these parameters are not provided, default values will be provided or extrapolated. For clarity, specifying at least
+which `èasyblock`` to use is a good idea.
 
 ### *Extensions*
 
 *Extensions* is the collective term we use for **additional software packages that can be installed
-on top of another software package**. Examples are *Python packages*, *R libraries* and *Perl modules*
-(can you tell why we had to come up with a different term?).
+on top of another software package**. Examples are *Python packages*, *R libraries* and *Perl modules*.
 
 Extensions can be installed in different ways:
 
 * stand-alone, as a separate installation on top of one or more other installations;
 * as a part of a bundle of extensions that collectively form a separate installation;
-* or as, well, an *extension* to a specific installation to yield a "batteries included"
+* or as an *extension* to a specific installation to yield a "batteries included"
   type of installation (for examples by adding a bunch of Python packages from PyPI into
   a Python installation);
 
 ### *Dependencies*
 
-A *dependency* is a common term in the context of software, which we probably don't need to define at length,
-but here it goes anyway: it refers to **a software package that is either strictly required by other software, 
-or that can be leveraged to enhance other software** (to support specific features for example).
+A *dependency* is a common term in the context of software. It refers to **a software
+package that is either strictly required by other software, or that can be leveraged to
+enhance other software** (for example to support specific features).
 
-There are multiple types of dependencies:
+EasyBuild supports multiple types of dependencies:
```

Comment:
That's not fully correct, since we don't currently have a separate easyconfig parameter for link dependencies.... This change should be rolled back?

Reply:
Ah, right. Hm. I'm not sure to what extent we should be teaching people compiler details that aren't relevant to EasyBuild, but I see your point. If we're going that route, I would suggest:
"There are three main types of dependencies for computer software:"

Reply:
OK!